Library/OcAppleKernelLib/CpuidPatches.c

```diff
@@ -1654,7 +1654,28 @@ PatchProvideCurrentCpuInfo (
 
     tscFreqValue    = CpuInfo->CPUFrequency;
     tscFCvtt2nValue = DivU64x64Remainder ((1000000000ULL << 32), tscFreqValue, NULL);
-    tscFCvtn2tValue = DivU64x64Remainder ((1000000000ULL  << 32), tscFCvtt2nValue, NULL);
+    tscFCvtn2tValue = DivU64x64Remainder ((1000000000ULL << 32), tscFCvtt2nValue, NULL);
+
+  } else if (CpuInfo->ExtFamily == 0xB) {
+
```

Comment:
Why cannot we just add a 100 MHz fallback to the upper if?

Reply:
Can do. I had it split for testing incase further modification was required. I'll have it tested to ensure compatibility.